### PR TITLE
YEA-164 Mainnet deployment

### PR DESCRIPTION
**Pull request by tehnalogos**

### CODE REVIEW

**Review by Ledespana — APPROVED** [review]

File: solidity/hardhat.config.ts [hunk 1]
```diff
@@ -11,10 +11,15 @@ const config: HardhatUserConfig = {
     // public LUKSO Testnet
     networks: {
       luksoTestnet: {
-        url: "https://rpc.testnet.lukso.network",
+        url: "https://lukso-testnet.rpc.thirdweb.com",
         chainId: 4201,
         accounts: [process.env.EOA_PRIVATE_KEY as string] // your private key here
       },
+      luksoMain: {
+        url: "https://lukso.rpc.thirdweb.com",
```
Comment (democrazyfi) [inline solidity/hardhat.config.ts:19]: https://docs.lukso.tech/networks/mainnet/parameters/  says requires an api key?

File: solidity/hardhat.config.ts [hunk 2]
```diff
@@ -11,10 +11,15 @@ const config: HardhatUserConfig = {
     // public LUKSO Testnet
     networks: {
       luksoTestnet: {
-        url: "https://rpc.testnet.lukso.network",
+        url: "https://lukso-testnet.rpc.thirdweb.com",
```
Comment (democrazyfi) [inline solidity/hardhat.config.ts:14]: how is it different to official one?

Comment (tehnalogos) [reply solidity/hardhat.config.ts:14]: Screenshot 2024-02-15 at 11 29 41 AM
Lukso themselves recommends using one of the other providers

File: frontend/components/wallet/WalletProvider.tsx [hunk 3]
```diff
@@ -100,14 +101,13 @@ export const WalletProvider: React.FC<Props> = ({ children }) => {
         if (!accounts.length) {
           throw new Error('No user accounts found');
         }
-        console.log('Connected with', accounts[0]);
-        // Update state and localStorage with the first account address.
-        setAccount(accounts[0]);
+        const connectedAccount = accounts[0];
+        console.log('Connected with', connectedAccount);
```
Comment (hosainnet) [inline frontend/components/wallet/WalletProvider.tsx:105]: needed?

File: frontend/components/wallet/WalletProvider.tsx [hunk 4]
```diff
@@ -116,10 +116,11 @@ export const WalletProvider: React.FC<Props> = ({ children }) => {
           siweMessage,
           signature as string
         );
+        console.log("The account's Main Controller address is:", signerAddress);
```
Comment (hosainnet) [inline frontend/components/wallet/WalletProvider.tsx:119]: log here too

File: frontend/components/wallet/WalletProvider.tsx (hunk 3 above)
Comment (democrazyfi) [inline frontend/components/wallet/WalletProvider.tsx:105]: console log needed?

File: frontend/components/wallet/WalletProvider.tsx (hunk 4 above)
Comment (democrazyfi) [inline frontend/components/wallet/WalletProvider.tsx:119]: here too

File: frontend/constants/networks.ts [hunk 5]
```diff
@@ -10,14 +11,16 @@ export interface Network {
...
   mainnet: {
     chainId: 42,
+    name: 'LUKSO',
     symbol: 'LYX',
     rpcUrl: 'https://rpc.lukso.gateway.fm',
     explorerURL: 'https://explorer.execution.mainnet.lukso.network',
-    universalGraveForwarder: '0xa5467dfe7019bf2c7c5f7a707711b9d4cad118c8',
-    lsp1UrdVault: '',
+    universalGraveForwarder: '0x433908ce6457b302a6452257Bc40e466d95c78E9',
+    lsp1UrdVault: '0x9292dAf1cdc3d03a1A0BbD4B3319C49A3B91d703',
```
Comment (democrazyfi) [inline frontend/constants/networks.ts:19]: Where did you find this?

Comment (democrazyfi) [reply frontend/constants/networks.ts:19]: Oh I see, you deployed in solidity/contracts/LSP1UniversalReceiverDelegateVault.sol ?

**Review by democrazyfi — APPROVED** [review]
Minor comments but lgtm

File: frontend/constants/networks.ts (hunk 5 above)
Comment (tehnalogos) [reply frontend/constants/networks.ts:19]: @democrazyfi yes ser :)

File: frontend/components/wallet/WalletProvider.tsx (hunk 4 above)
Comment (tehnalogos) [reply frontend/components/wallet/WalletProvider.tsx:119]: @democrazyfi specifically left it in so we can monitor behavior for a bit

**Pull request merged**